main.cpp

```diff
@@ -217,345 +256,337 @@ struct AlienDetectionAgency //this is slightly revised from 1d
+    void initBoard();
+};
+
+int Board::detectObjectType(std::string object, int objHeight)
+{
+    //detect if object lower than min height - returns 0 otherwise
+    int minHeight = 5;
+    height = objHeight;
+
+    if (height <= minHeight)
     {
-        //1) Detect object on surface
-        if (objHeight <= 0.0f)
+        if(object == "marker")
         {
-            if(object == "marker")
-            {
-                return 1;
-            }
-            //2) Detect hand on surface
-            else if(object == "Lhand")
-            {
-                return 2;
-            }
-            //2) Detect hand on surface
-            else if(object == "Rhand")
-            {
-                return 3;
-            }     
+            return 1;
         }
+        else if(object == "Lhand")
+        {
+            return 2;
+        }
+        else if(object == "Rhand")
+        {
+            return 3;
+        }     
     }
-    //3) Reposition in world space
-    void newPosition(int x, int y, int z);
- };
+    return 0;
+}
+
+void Board::newPosition(int x, int y, int z)
+{
+    worldX = x;
+    worldY = y;
+    worldZ = z;
+}
 
+void Board::initBoard()
+{
 
-/*
-Thing 7) Fiducial Marker
-5 properties:
-    1) Value
-    2) Position horizontal
-    3) Position Vertical
-    4) Rotation
-    5) Elevation
-3 things it can do:
-    1) send position
-    2) send value
-    3) send rotation
- */
+}
 
 struct FiducialMarker
 {
-    //1) Value (ID)
     std::string markerID= "marker01";
-    //2) Position horizontal
     int posX = 10;
-    //3) Position Vertical
     int posY = 100;
-    //4) Rotation
     int rotation = 0;
-    //5) Elevation
-    float elevation = 0.0f;
-
-    //1) send position
-    void sendMarkerPos( int X,int Y );
-    //2) send value
-    void sendMarkerID(std::string ID);
-    //3) send rotation
-    void sendMarkerRot(int rot);
-    //or really you just do all data in one call (avoids race condition)
-    void sendMarkerData(std::string ID, int X, int Y, float elev, int rot);
+    int elevation = 0.0f;
+
+    void setMarkerPos( int X,int Y );
+    void setMarkerID(std::string ID);
+    void setMarkerRot(int rot);
+    void setMarkerData(std::string ID, int X, int Y, int elev, int rot);
 };
 
-/*
-Thing 8) Audio Engine
-5 properties:
-    1) Master Volume
-    2) Section
-    3) Active instrument count
-    4) Active Effect count
-    5) External Input volume
-3 things it can do:
-    1) produce audio from instruments
-    2) apply effects to instruments
-    3) record sounds 
-*/
+void FiducialMarker::setMarkerPos(int x, int y)
+{
+    posX = x;
+    posY = y;
+}
+
+void FiducialMarker::setMarkerID(std::string ID)
+{
+    markerID = ID;
+}
+
+void FiducialMarker::setMarkerRot(int rot)
+{
+    rotation = rot;
+}
+
+void FiducialMarker::setMarkerData(std::string ID, int X, int Y, int elev, int rot)
+{
+    markerID = ID;
+    posX = X;
+    posY = Y;
+    elevation = elev;
+    rotation = rot;
```

Comment:
variable names need to start with a lowercase letter, even if they're single-letter names.
- [ ] revise